stream.go

```diff
@@ -0,0 +1,164 @@
+package main
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strconv"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/google/uuid"
+	"github.com/hashicorp/golang-lru/simplelru"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/rs/zerolog"
+	tmrpcclient "github.com/tendermint/tendermint/rpc/client"
+	rpchttp "github.com/tendermint/tendermint/rpc/client/http"
+	tmjsonclient "github.com/tendermint/tendermint/rpc/jsonrpc/client"
+	tmtypes "github.com/tendermint/tendermint/types"
+)
+
+var (
+	started                = false
+	queryEventBankTransfer = fmt.Sprintf(
+		"%s='%s' AND %s='%s'",
+		tmtypes.EventTypeKey,
+		tmtypes.EventTxValue,
+		"message.action",
+		"/cosmos.bank.v1beta1.MsgSend",
+	)
+	queryInterval = 100 * time.Millisecond
```

Comment:
Is this too quick? Our block times are higher than this and metrics are scraped every 30 seconds